stroeer/core/v1/article.proto

```diff
@@ -29,12 +29,8 @@ message Article {
   // Subtype of the article. For `CONTENT_TYPE_ARTICLE` this field holds a sub type, for others like `CONTENT_TYPE_GALLERY` it may not.
   ContentSubType sub_type = 3;
 
-  // References to get access to the article on different technical platforms like
-  // canonical web or AMP (required).
-  stroeer.core.v1.References references = 4;
-
   // Hierarchical section tree information of the article (required).
-  stroeer.core.v1.SectionTree section_tree = 5;
+  stroeer.core.v1.Category section_tree = 5;
```

Comment:
I still like `section_tree` as field name.

Reply:
this does not matter on the (protobuf) wire, anyway.